src/content/reference/web-interface/app-catalog.md

```diff
@@ -0,0 +1,46 @@
++++
+title = "The Giant Swarm App Catalog in The Web Interface"
+description = "What the Giant Swarm App Catalog looks like on our Web Interface and how to use it."
+date = "2019-07-05"
+layout = "subsection"
+weight = 10
++++
+
+# The Giant Swarm App Catalog in The Web Interface
+
+This page will give you an overview of how to do some common tasks related to the
+Giant Swarm App Catalog using our Web Interface.
+
+Go here for an [overview of the Giant Swarm App Catalog](/basics/app-catalog/) instead.
```

Comment:
I suggest - 
For an overview of the Giant Swarm App Catalog, go here instead

Reply:
It's structured like that because I want that last bit to be the link that people click on.

```
Go here for an [overview of the Giant Swarm App Catalog].  
```

makes more sense if the part in brackets is the link compared to:

```
For an [overview of the Giant Swarm App Catalog], go here instead.
```

The reason is I want to avoid linking words like "go here". As a general best practice links should be descriptive of what they will link too.